gordo_components/dataset/datasets.py

```diff
@@ -16,24 +16,23 @@
 logger = logging.getLogger(__name__)
 
 
-class TimeSeriesDataset(GordoBaseDataset):
+class RawTimeSeriesDataset(GordoBaseDataset):
```

Comment:
Very minor thing, but can you put the `RawTimeSeriesDataset` below the `TimeSeriesDataset` ? Then maybe the diff will be less dumb

Reply:
No can do, daddy-o. Then `RawTimeSeriesDataset` if referenced before assignment to the inheritance in `TimeSeriesDataset` :man_shrugging: